frontend/src/modules/entitydetails/components/EntityDetails.test.js

```diff
@@ -107,6 +107,43 @@ describe('<EntityDetailsBase>', () => {
         expect(wrapper.text()).toContain('Metadata');
         expect(wrapper.text()).toContain('Editor');
     });
+
+    it('shows failed checks for approved (or fuzzy) translations with errors or warnings', () => {
+        const wrapper = createShallowEntityDetails();
+
+        wrapper.setProps({
+            selectedEntity: {
+                pk: 2,
+                original: 'something',
+                translation: [{
+                    approved: true,
+                    string: 'quelque chose',
+                    errors: ['Error1'],
+                }],
+            },
+        });
+
+        expect(editor.actions.updateFailedChecks.calledOnce.toBeTruthy());
```

Comment:
```js
expect(editor.actions.updateFailedChecks.calledOnce).toBeTruthy();
```

`toBeTruthy` is a property of the `expect` framework. :-)

Reply:
For this to work, you'll need to mock those two actions though. See for example: https://github.com/mozilla/pontoon/blob/master/frontend/src/modules/entitydetails/components/EntityDetails.test.js#L115

Reply:
Thanks :) Updated, but the 2nd test is still failing.